src/Components/ArticleExpanded/ArticleExpanded.js

```diff
@@ -1,10 +1,88 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
+import styled from "styled-components";
+import {darkTheme} from "../../theme/globalStyle";
+
+const {
+  primaryPurple,
+  primaryBlue,
+  secondaryTeal,
+  background,
+  textColor,
+  error
+} = darkTheme;
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  height: 100%;
+  background-color: ${background};
+  padding: 3%;
+
+  img {
+    width: 100%;
+  }
+
+  h2 {
+    font-size: 1.6em;
+    padding: 3%;
+  }
+
+  p {
+    padding: 3%;
+  }
+
+  button {
+    width: 40%;
+    background-color: ${secondaryTeal};
+    margin: 5%;
+    border: none;
+    padding: 3%;
+    font-size: .8em;
+    border-radius: 5px;
+  }
+
+  button:hover {
+    background-color: ${primaryBlue};
+    color: #FFFFFF;
+  }
+`
+const ImgContainer = styled.section`
+  width: 90%;
+  align-self: center;
+`
+
+const BtnContainer = styled.section`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+
+`
 
 const ArticleExpanded = (props) => {
+
+  const [isSaved, setIsSaved] = useState(false);
+
+  const handleClick = () => {
+    setIsSaved(!isSaved);
+    props.updateSavedArticles(props.clickedArticle);
+  }
+
   return (
-  <div>
+  <Wrapper>
+    <h2>{props.clickedArticle.title}</h2>
+    <ImgContainer>
+      <img src={props.clickedArticle.image}/>
+    </ImgContainer>
     <p>{props.clickedArticle.description}</p>
-  </div>
+    <BtnContainer>
+      <button>Go to Article</button>
```

Comment:
I think we can make this button functional by interpolating the URL into an <a> tag